src/test/regress/expected/pg14.out

```diff
@@ -143,5 +143,50 @@ DETAIL:  on server postgres@localhost:xxxxx connectionId: xxxxxxx
 reindex(TABLESPACE test_tablespace1) index idx;
 ERROR:  tablespace "test_tablespace1" does not exist
 set citus.log_remote_commands to off;
+-- test column compression propagation in distribution
+SET citus.shard_replication_factor TO 1;
+CREATE TABLE col_compression (a TEXT COMPRESSION pglz, b TEXT);
+SELECT create_distributed_table('col_compression', 'a', shard_count:=4);
+ create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+SELECT attname || ' ' || attcompression AS column_compression FROM pg_attribute WHERE attrelid::regclass::text LIKE 'col\_compression%' AND attnum > 0 ORDER BY 1;
+ column_compression
+---------------------------------------------------------------------
+ a p
+ b
+(2 rows)
+
+SELECT result AS column_compression FROM run_command_on_workers($$SELECT ARRAY(
+SELECT attname || ' ' || attcompression FROM pg_attribute WHERE attrelid::regclass::text LIKE 'pg14.col\_compression%' AND attnum > 0 ORDER BY 1
+)$$);
+   column_compression
+---------------------------------------------------------------------
+ {"a p","a p","b ","b "}
+ {"a p","a p","b ","b "}
+(2 rows)
+
+-- test column compression propagation in rebalance
+INSERT INTO col_compression SELECT 'a', 'b' FROM generate_series(1, 1000000);
```

Comment:
how long this test take? If it is more than ~100 msecs, I'd suggest we simply do `citus_move_shard_placement` without loading any data